.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and test
+
+on:
+  push:
+    branches: [ "develop", "develop_1.x" ]
```

Comment:
we don't usually run the CI on push but if there's no penalty I'm fine with it.

Reply:
Yeah, makes sense... I just took the default Xcode Action template and modified it. I suppose once `develop` and `develop_1.x` are protected this is somewhat academic (since we would only have `pull_request` coming in). I can remove it 👍